.github/workflows/tests.yaml

```
@@ -19,93 +19,39 @@ jobs:
     - name: Run lint & unit tests
       run: tox
 
-  func-test:
-    name: Functional test with MicroK8s
-    runs-on: ubuntu-latest
-
+  integration-test:
+    name: VSphere Integration Test
+    runs-on: self-hosted
+    timeout-minutes: 90
```

Add this to prevent integration tests until the unit tests and linting are done

```suggestion
    timeout-minutes: 90
    needs: 
      - lint-and-unit-tests
```

---

this will cut down on using the integration tests framework when you KNOW you're gonna have to push more changes to fix the linting/unit testing

---

good thinkin